src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-metrics.mdx

```diff
@@ -0,0 +1,1131 @@
+    </td>
+    </tr>
+
+            <tr>
+      <td>
+        `value`
+      </td>
+
+            <td>
+        `double`
+      </td>
+
+      <td>
+        Required. The value of the metric.
+      </td>
+    </tr>
+
+                <tr>
+      <td>
+        `count`
+      </td>
+
+            <td>
+        `int`
+      </td>
+
+      <td>
+        Required. The number of times the metric was recorded.
+      </td>
+    </tr>
+
+                    <tr>
+      <td>
+        `totalValue`
+      </td>
+
+            <td>
+        `double`
+      </td>
+
+      <td>
+        Required. The total value of the metric.
+      </td>
+    </tr>
+
+                    <tr>
+      <td>
+        `exclusiveValue`
+      </td>
+
+            <td>
+        `double`
+      </td>
+
+      <td>
+        Required. The exclusive value of the recording; for example, if the total value contains measurements accounted for elsewhere.
+      </td>
+    </tr>
+
+                        <tr>
+      <td>
+        `countUnit`
+      </td>
+
+            <td>
+        `MetricUnit`
+      </td>
+
+      <td>
+        Required. Unit of measurement for the metric count, including `PERCENT`, `BYTES`, `SECONDS`, `BYTES_PER_SECOND`, or `OPERATIONS`.
+      </td>
+    </tr>
+
+                        <tr>
+      <td>
+        `valueUnit`
+      </td>
+
+            <td>
+        `MetricUnit`
+      </td>
+
+      <td>
+        Required. Unit of measurement for the metric value, including `PERCENT`, `BYTES`, `SECONDS`, `BYTES_PER_SECOND`, or `OPERATIONS`.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+## Examples [#examples]
+
+Here's an example of creating simple custom metrics:
+
+### Java [#java]
+
+```java
+NewRelic.recordMetric("Custom Metric Name","MyCategory", 1.0);
+NewRelic.recordMetric("Login Auth Metric", "Network", 1.0);
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
+
+Here's an example of creating a custom metric for agent start:
+
+### Java [#java]
+
+```java
+NewRelic.recordMetric("Agent start", "Lifecycle");
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
+
+Here's an example of the same metric above, but with elapsed time value added:
+
+### Java [#java]
+
+```java
+NewRelic.recordMetric("Agent start", "Lifecycle", 10.11f);
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
+
+Here's an example of the same metric above, but with two counts of elapsed time and exclusive time:
+
+### Java [#java]
+
+```java
+NewRelic.recordMetric("Agent Start", "lifecycle", 2, 10.11, 1.23)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
+
+Here's an example of the same metric above, but with five counts of elapsed time, exclusive time, and a unit value parameter:
+
+### Java [#java]
+
+```java
+NewRelic.recordMetric("Agent start", "Lifecycle", 5, 10.11, 1.23, MetricUnit.OPERATIONS, MetricUnit.SECONDS);
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
```suggestion
```kotlin
    NewRelic.recordMetric("Agent start", "Lifecycle", 5, 10.11, 1.23, MetricUnit.OPERATIONS, MetricUnit.SECONDS)

```
```